server/requirements-common.txt

```diff
@@ -6,4 +6,7 @@ gunicorn==20.0.4
 transformers==4.26.1
 huggingsound==0.1.6
 jiwer==2.5.1
-metaphone==0.6
\ No newline at end of file
+metaphone==0.6
+PyMySQL
```

Comment:
Can you add a specific version for each module?

Reply:
Fixed.